.github/workflows/vulnerability-scanner-generate-database.yml

```diff
@@ -73,11 +99,16 @@ jobs:
 
       - name: Vulnerability scanner - Uploading file to S3
         run: |
```

Comment:
Nitpick: The condition can be moved here, [see example](https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-inputs-context-in-a-reusable-workflow)

Reply:
I've tried to skip the steps but the **if** condition seems to work only for jobs.